### ISSUE TITLE
(Suggestion) Eye Protection Filter

### ISSUE THREAD

**Issue by Minionguyjpro** [windows-server-2003/ThirdTube#96]
Add a eye protection filter like Windows has that gives a yellow filter to the screen, but then for ThirdTube so that we can watch video's on a way that your eyes need to do less.

**Comment by CSab6482** [comment]
There is a [modified version of Luma](https://www.gamebrew.org/wiki/Unofficial_Luma_3DS) that has this filter option enabled and I find it very useful. You access it with the same L+select+D-Pad down button combination, and it works on all 3DS software.

**Comment by Minionguyjpro** [comment]
> There is a [modified version of Luma](https://www.gamebrew.org/wiki/Unofficial_Luma_3DS) that has this filter option enabled and I find it very useful. You access it with the same L+select+D-Pad down button combination, and it works on all 3DS software.

That is true, but it is easier to have an filter builded in, right?

**Comment by CSab6482** [comment]
I suppose, but what I meant to get at is that [Redshift for the 3DS exists](https://www.gamebrew.org/wiki/Redshift_3DS), and this is simply one way of getting it installed. I feel like if this feature were implemented into ThirdTube, then you would be limited to using it in ThirdTube instead of across the entire OS.

**Comment by Minionguyjpro** [comment]
> 

I think i understand you. So eye protection filter is limited as the application itself makes the pixels like yellow tinted, but the OS can really do that for the real display.

**Comment by Minionguyjpro** [comment]
> I suppose, but what I meant to get at is that [Redshift for the 3DS exists](https://www.gamebrew.org/wiki/Redshift_3DS), and this is simply one way of getting it installed. I feel like if this feature were implemented into ThirdTube, then you would be limited to using it in ThirdTube instead of across the entire OS.

And this is for an custom Luma3DS build, which i can't find of a boot.3dsx and boot.firm, but only source code. (Also i do have custom Luma3DS but from a different developer.)

**Comment by CSab6482** [comment]
Download the .7z file from [here](https://dlhb.gamebrew.org/3dshomebrews/lumanut.7z), and then extract it. Then, extract the `luma1030_QuickSwitchers - NightLight .zip` file, and you should get the `boot.firm` file.

**Comment by Minionguyjpro** [comment]
> Download the .7z file from [here](https://dlhb.gamebrew.org/3dshomebrews/lumanut.7z), and then extract it. Then, extract the `luma1030_QuickSwitchers - NightLight .zip` file, and you should get the `boot.firm` file.

OK, i understand, but i don't want to switch my Luma3DS version as it just works better and is newer.

**Comment by windows-server-2003** [comment]
I haven't used the color filter integrated in the official luma3ds, but doesn't it work as a blue light filter?

**Comment by Minionguyjpro** [comment]
> I haven't used the color filter integrated in the official luma3ds, but doesn't it work as a blue light filter?
> 

@windows-server-2003 I also don't have the original version of Luma3DS, but the version of Nanquitas. And also I just tested and there are some filters.

https://i.imgur.com/gZ1hrQ8.jpg

**Comment by CSab6482** [comment]
These are the exact filter options I see on my modified build, so I guess I can switch back to stock Luma! Since Luma has these blue light filters, is this ticket still necessary?

**Comment by Minionguyjpro** [comment]
> These are the exact filter options I see on my modified build, so I guess I can switch back to stock Luma! Since Luma has these blue light filters, is this ticket still necessary?

This is not stock Luma3DS as i said.

**Comment by CSab6482** [comment]
Yes but the filter options appear to be in your custom build as well. Do they work as blue light filters?

**Comment by Minionguyjpro** [comment]
> Yes but the filter options appear to be in your custom build as well. Do they work as blue light filters?

Yes and no. There is a cold filter, which makes it a little bit more like-cold. But most of them, makes it yellow.

**Comment by ZippyZolton** [comment]
I have had stock Luma for over a year and I've always had the filter settings in the picture above

**Comment by Minionguyjpro** [comment]
Yeah, I think it's the same in both the Luma builds, but now they are one since Nanquitas' version got merged.
